api/topology/src/test/java/quarks/test/topology/PlumbingTest.java

```diff
@@ -601,4 +602,66 @@ public void testParallel() throws Exception {
 //            actDuration < 0.5 * expMinSerialDuration);
 //    }
 
-}
+    @Test
+    public void testGate() throws Exception {
+        // Timing variances on shared machines can cause this test to fail
```

Comment:
Does this "timing variances" failure condition actually happen?  I can't imagine how it could.  Same with 2nd test case (though may have to increase test-tmo beyone 1sec. for that one).